MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -531,42 +533,44 @@ private List<NameValuePair> ReadValueArray(XmlReader reader)
         private void ProcessPairs(string key, List<NameValuePair> pairs, MediaInfo info)
         {
             List<BaseItemPerson> peoples = new List<BaseItemPerson>();
+            var distinctPairs = pairs.Select(p => p.Value)
+                    .Where(i => !string.IsNullOrWhiteSpace(i))
+                    .Trimmed()
+                    .Distinct(StringComparer.OrdinalIgnoreCase);
```

Comment:
The purpose of this is to remove duplicates in an input list as we sometimes get the same person twice.